modules/context/repo.go

```diff
@@ -308,11 +308,17 @@ func EarlyResponseForGoGetMeta(ctx *Context) {
 		ctx.PlainText(http.StatusBadRequest, "invalid repository path")
 		return
 	}
-	ctx.PlainText(http.StatusOK, com.Expand(`<meta name="go-import" content="{GoGetImport} git {CloneLink}">`,
+	res, err := vars.Expand(`<meta name="go-import" content="{GoGetImport} git {CloneLink}">`,
```

Comment:
We can just use `fmt.Sprintf` to replace them.

Reply:
The code was incorrect before, because it's not escaped correctly. A simple Sprintf is also incorrect.

If you want this change, then I can update the code with correctly escaped content.

https://github.com/golang/go/blob/2ebe77a2fda1ee9ff6fd9a3e08933ad1ebaea039/src/cmd/go/internal/vcs/discovery.go#L54

Reply:
See the latest commit